src/main/java/com/opentable/db/postgres/embedded/PreparedDbProvider.java

```diff
@@ -76,12 +76,36 @@ private synchronized PrepPipeline createOrFindPreparer(DatabasePreparer preparer
 
     /**
      * Create a new database, and return it as a JDBC connection string.
-     * No two invocations will return the same database.
+     * NB: No two invocations will return the same database.
      */
     public String createDatabase() throws SQLException
     {
```

Comment:
A fair question.

Reply:
@SegFaultAX because the code was imported from a codebase with different style tendencies, and updates have been in the "new style" rather than preserving, leading to a mix :/